modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/macros/TaxonomyVocabularyAPI.macro

```diff
@@ -228,6 +246,14 @@ definition {
 		return ${elementValue};
 	}
 
+	macro getKeywordsBySiteId {
```

Comment:
Just a kindly reminder that shall we rename this method to getKyworksByGrounpName?

Reply:
Yeah, you're right, I will edit it.